Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -390,7 +390,14 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
         metrics.transactionMetrics.forEach { transaction in
             if let startDate = transaction.fetchStartDate,
                let endDate = transaction.responseEndDate,
-               let path = transaction.request.url?.path {
+               var path = transaction.request.url?.path {
+                
+                if path.contains("graphql"), 
```

Comment:
Are we able to add a unit test for this?

Reply:
Sure, added: [df95e39](https://github.com/braintree/braintree_ios/pull/1349/commits/df95e39eb534c777de12ca45a50e2c8e3fd5d7fe)